oak_functions/client/java/src/AttestationClient.java

```diff
@@ -43,14 +40,14 @@ public void onNext(AttestedInvokeResponse response) {
                 try {
                     messageQueue.put(response);
                 } catch (Exception e) {
-                    logger.log(Level.WARNING, "Couldn't send server response to the message queue: {0}", e);
+                    System.out.printf("Couldn't send server response to the message queue: %s\n", e.toString());
```

Comment:
Is the call to `toString` really needed?

Reply:
Yes, in order to print it via `%s`.
IIUC Java cannot print raw Objects using `System.out.printf` (but please correct me if I'm wrong)

Reply:
I could answer this question a year ago. My java is very rusty now :D